flask_azure_oauth/__init__.py

```diff
@@ -9,6 +9,8 @@
 
 class FlaskAzureOauth(ResourceProtector):
     def __init__(self):
+        super().__init__()
```

Comment:
Have ResourceProtector initialize first, since the <sup>*</sup>Authlib ResourceProtector [now contains](https://github.com/lepture/authlib/blob/b8c2c29788a9394ae31dd267bc770fc891c009f7/authlib/oauth2/rfc6749/resource_protector.py#L15):
```
class ResourceProtector(object):
    def __init__(self):
        self._token_validators = {}
```
<sup>*In this case, the ResourceProtector is actually `flask_azure_oauth.resource_protector`, which wraps the Authlib ResourceProtector.</sup>